js/twitter-text.js

```diff
@@ -1172,23 +1114,6 @@
 
   var MAX_LENGTH = 140;
 
-  // Characters not allowed in Tweets
-  var INVALID_CHARACTERS = [
-    // BOM
-    fromCode(0xFFFE),
-    fromCode(0xFEFF),
-
-    // Special
-    fromCode(0xFFFF),
-
-    // Directional Change
```

Comment:
are these directional change characters getting lost in the refactor of INVALID_CHARS and INVALID_CHARACTERS ?